crates/air-lib/interpreter-interface/src/interpreter_outcome.rs

```diff
@@ -75,13 +75,26 @@ impl InterpreterOutcome {
             ));
         }
 
-        let call_requests = try_as_byte_vec(record_values.pop().unwrap(), "call_requests")?;
+        let call_requests = try_as_byte_vec(
```

Comment:
As marine doesn't keep info about which structure it was, we may pass `Vec<u8>` in the boundaries, simplifying the implementation.

Reply:
Implemented.